.travis.yml

```diff
@@ -19,3 +19,4 @@ notifications:
       - matthew@bitovi.com
     on_success: change
     on_failure: change
+script: npm run coverage:upload
```

Comment:
This means the guide tests are not running in Travis.  We need to be able to run all of these things as well: https://github.com/donejs/donejs/blob/cli-refactor/package.json#L10